modules/utility.py

```diff
@@ -468,28 +469,49 @@ async def _info(self, ctx, user: typing.Union[discord.Member, int]):
 
         else:
             puns = 0
+            activeStrikes = 0
+            totalStrikes = 0
             for pun in punsCol.sort('timestamp', pymongo.DESCENDING):
+                if pun['type'] == 'strike':
+                    totalStrikes += pun['strike_count']
+                    activeStrikes += pun['active_strike_count']
+
+                elif pun['type'] == 'destrike':
+                    totalStrikes -= pun['strike_count']
+
                 if puns >= 5:
-                    break
+                    continue
 
                 puns += 1
                 stamp = datetime.datetime.utcfromtimestamp(pun['timestamp']).strftime('%m/%d/%y %H:%M:%S UTC')
                 punType = config.punStrs[pun['type']]
-                if pun['type'] in ['clear', 'unmute', 'unban', 'unblacklist']:
+                if pun['type'] in ['clear', 'unmute', 'unban', 'unblacklist', 'destrike']:
+                    if pun['type'] == 'destrike':
+                        punType = f'Removed {pun["strike_count"]} Strike{"s" if pun["strike_count"] > 1 else ""}'
+
                     punishments += f'- [{stamp}] {punType}\n'
 
                 else:
+                    if pun['type'] == 'strike':
+                        punType = f'{pun["strike_count"]} Strike{"s" if pun["strike_count"] > 1 else ""}'
+
                     punishments += f'+ [{stamp}] {punType}\n'
 
             punishments = f'Showing {puns}/{punsCol.count()} punishment entries. ' \
-                f'For a full history including responsible moderator, active status, and more use `{ctx.prefix}history @{str(user)}` or `{ctx.prefix}history {user.id}`' \
+                f'For a full history including responsible moderator, active status, and more use `{ctx.prefix}history {user.id}`' \
                 f'\n```diff\n{punishments}```'
+
+            if totalStrikes:
+                embed.description = embed.description + f'\nUser currently has {activeStrikes} active strike{"s" if activeStrikes > 1 or activeStrikes < 1 else ""} ({totalStrikes} in total)'
```

Comment:
`activeStrikes > 1 or activeStrikes < 1` can be replaced with `activeStrikes != 1`

Reply:
Resolved in 79046a7